src/components/Homepage/LendByCategory/FeaturedLoansCarousel.vue

```diff
@@ -1,14 +1,23 @@
 <template>
 	<div>
 		<h3>
-			<a :href="currentSlideCategory.url">
+			<a
+				:href="currentSlideCategory.url"
+				v-kv-track-event="[
+					'homepage',
+					'click-hero-category',
+					cleanCategoryLink(currentSlideCategory),
+					'true']"
```

Comment:
Please remove 'true' from the 4th parameter of all of these. This was legacy code I believe, this parameter currently needs to be either a number or not present.